proposals/interface-types/working-notes/choices/webidl.md

```diff
@@ -0,0 +1,260 @@
+designers.
+
+In particular,
+
+>Interface Types are intended to enable WebAssembly modules to interoperate
+>across ownership boundaries.
+
+The enriched type system that is defined by the Interface Types proposal has
+some of the hallmarks of an IDL (Interface Definition Language). However, there
+remain several questions, which this note aims to address:
+
+* What are the design considerations that lie behind the specification of the
+  Interface Types language of types?
+* Why not WebIDL?
+* Are we intending to support interoperability between different programming
+  languages?
+* Given the importance of the web platform, what is the relationship between the
+  Interface Types and WebIDL?
+  
+### Supporting Interoperability Across Ownership Boundaries
+
+Interoperating across ownership boundaries has different expectations than when
+within a shared domain. In particular, there more limited expectations for
+sharing, architecture, and trust.
+
+The term 'ownership boundary' bears further explanation. Informally, we can say
+that we are crossing an ownership boundary when, for example, my code is calling
+your code and I want to limit the extent that I have to trust you.
+
+By restricting the intended scope to interoperability across ownership
+boundaries the design of Interface Types can sidestep the issues that cause
+stumbling blocks for the general inter-language interopability.
+
+### Interoperability via Function Calls
+
+Interoperability between different systems is arguably a requirement of all
+modern applications. However, one of the differentiating aspects of
+interoperability as it applies to WebAssembly is the role of the function
+call. Unlike interoperability expressed at the level of web services (say),
+using a foreign capability in WebAssembly is most naturally expressed as a
+function call.
+
+I.e., invoking capabilities does not naturally imply sending messages,
+serializing data, network connections, or even crossing a process boundary: the
+invoked capability may even be present in the same thread.
+
+This view of interoperability informs much of the style of the Interface Types
+proposal. In particular, this proposal does not include a serialization
+format.[^Although there is a binary encoding, its role is to support the
+publishing and loading of WebAssembly modules; not to support function
+interaction.] Indeed, it is expected that no serialization is required when
+invoking a function via Interface Types' adapters.
+
+### Language Specific Module Systems
+
+Interface Types are _not_ intended to replacement language specific module
+systems. Nor are Interface Types intended to address the general inter-language
+interoperability problem.
+
+Indivual languages will often have their own techniques aimed at allowing
+importing and exporting of modules. For example, C/C++ modules would continue to
+be linked together into single WebAssembly modules.
+
+## WebIDL
```

Comment:
I appreciate that the content above this is slimmed down from previous versions, but I think it could be slimmed further.

If you were to remove the previous three sections, then the document would dive immediately into the heart of the matter after the introduction. Given that the purpose of this document is to explain "why not WebIDL", getting to the point seems valuable to me.